ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -0,0 +1,369 @@
+          </Button>}
+        </div>
+        <div style={
+          (domain === DomainType.PERSON && type !== CriteriaType.AGE)
+            ? {marginBottom: '3.5rem'}
+            : {height: 'calc(100% - 3.5rem)'}
+        }>
+          {domain === DomainType.PERSON ? <div style={{flex: 1, overflow: 'auto'}}>
+              <Demographics
+                count={count}
+                criteriaType={type}
+                select={this.addSelection}
+                selectedIds={selectedIds}
+                selections={selections}/>
+            </div>
+            : <React.Fragment>
+              {loadingSubtree && <SpinnerOverlay/>}
+              <div style={loadingSubtree ? {height: '100%', pointerEvents: 'none', opacity: 0.3} : {height: '100%'}}>
+                {/* Tree View */}
+                <div style={this.searchContentStyle('tree')}>
+                  {hierarchyNode && <CriteriaTree
+                      autocompleteSelection={autocompleteSelection}
+                      back={this.back}
+                      groupSelections={groupSelections}
+                      node={hierarchyNode}
+                      scrollToMatch={this.setScroll}
+                      searchTerms={treeSearchTerms}
+                      select={this.addSelection}
+                      selectedIds={selectedIds}
+                      selectOption={this.setAutocompleteSelection}
+                      setAttributes={this.setAttributes}
+                      setSearchTerms={this.setTreeSearchTerms}/>}
+                </div>
+                {/* List View (using duplicated version of ListSearch) */}
+                <div style={this.searchContentStyle('list')}>
+                  <ListSearchV2 hierarchy={this.showHierarchy}
+                              searchContext={searchContext}
+                              select={this.addSelection}
+                              selectedIds={selectedIds}
+                              setAttributes={this.setAttributes}/>
+                </div>
+                {/**
+                 Attributes Page - This will no longer be rendered here in the future, leaving temporarily for reference
+                 TODO Remove once AttributesPage is moved to the sidebar with RW-4595
+                 **/}
+                <div style={this.searchContentStyle('attributes')}>
+                  {!!attributesNode && <AttributesPage
+                      close={this.back}
+                      node={attributesNode}
+                      select={this.addSelection}/>}
+                </div>
+              </div>
+            </React.Fragment>}
+          {type === CriteriaType.AGE && <div style={styles.footer}>
+            <Button style={{height: '1.5rem', margin: '0.25rem 0.5rem'}}
+                    type='link'
+                    onClick={closeSearch}>
+              Cancel
+            </Button>
+            <Button style={{height: '1.5rem', margin: '0.25rem 0.5rem'}}
```

Comment:
Optional: We can create a style object for button as its same (as of now)

Reply:
Done